hazelcast/src/main/java/com/hazelcast/util/ServiceLoader.java

```diff
@@ -300,36 +292,36 @@ public boolean equals(Object o) {
             }
 
             URLDefinition that = (URLDefinition) o;
-
             if (uri != null ? !uri.equals(that.uri) : that.uri != null) {
                 return false;
             }
-
             return true;
         }
 
         @Override
         public int hashCode() {
-            int result = uri != null ? uri.hashCode() : 0;
-            return result;
+            return uri != null ? uri.hashCode() : 0;
```

Comment:
flavor.

I find '==' easier to read than '!=', so I prefer to rewrite the code to:

return uri == null?0:uri.hashCode()

This is purely a flavor based thing.